test/e2e/testcase.yaml

```diff
@@ -0,0 +1,100 @@
+name: autok3s
+param:
+  server: http://autok3s:8080
+spec:
+  kind: HTTP
+api: http://autok3s:8080/v1
+items:
+- name: createAlibabaCredential
+  request:
+    api: /credentials
+    method: POST
+    header:
+      Content-Type: application/json
+    bodyFromFile: data/credential-alibaba.json
+  expect:
+    statusCode: 201
+- name: createAwsCredential
+  request:
+    api: /credentials
+    method: POST
+    bodyFromFile: data/credential-aws.json
+  expect:
+    statusCode: 201
+- name: createGoogleCredential
+  request:
+    api: /credentials
+    method: POST
+    bodyFromFile: data/credential-google.json
+  expect:
+    statusCode: 201
+- name: createTencentCredential
+  request:
+    api: /credentials
+    method: POST
+    bodyFromFile: data/credential-tencent.json
+  expect:
+    statusCode: 201
+
+- expect:
+    statusCode: 201
+  name: createK3dCluster
+  request:
+    api: /clusters
+    bodyFromFile: data/cluster-k3d.json
+    method: POST
+- expect:
+    statusCode: 201
+  name: createNativeCluster
+  request:
+    api: /clusters
+    bodyFromFile: data/cluster-native.json
+    method: POST
```

Comment:
no expect assert block?

Reply:
the default assert is checking if the status code is `200`

Reply:
It is best to display the declaration to increase readability